### PR TITLE
Implement Gradient trait for RadialGradient

### PR DESCRIPTION
**Pull request by Jayshua**
I could be wrong about this, maybe I'm just missing something. Shouldn't the Gradient trait be implemented for RadialGradient?

### CODE REVIEW

**Comment by EPashkin** [comment]
:+1:

File: src/patterns.rs [hunk 1]
```diff
@@ -245,6 +245,8 @@ impl RadialGradient {
     }
 }
 
+impl Gradient for RadialGradient{}
+
```
Comment (GuillaumeGomez) [inline src/patterns.rs]: Extra empty new line.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Comment by gkoz** [comment]
:+1:

**Comment by GuillaumeGomez** [comment]
So everyone is ok with your change. Just remove the extra empty line, squash the extra commit and we're all good!

**Comment by Jayshua** [comment]
Not that I have a problem with removing the extra line, but I was trying to match the formatting [already there](https://github.com/gtk-rs/cairo/blob/72857af40478d37a2159177fafe75ce10edfeb6f/src/patterns.rs#L247-L248) and just under the [LinearGradient code](https://github.com/gtk-rs/cairo/blob/72857af40478d37a2159177fafe75ce10edfeb6f/src/patterns.rs#L220-L223). Are you sure it shouldn't be there?

**Comment by GuillaumeGomez** [comment]
Not sure about this...

@gkoz: ?

**Comment by gkoz** [comment]
I'll support @Jayshua on this because following the apparent style is good and it feels uncool to punish a contributor for someone's peculiar choice of style some years ago ;)

**Comment by GuillaumeGomez** [comment]
@gkoz: I just thought it was a typo. Don't want to punish anyone haha.

@Jayshua: Thanks for the PR and sorry for the delay!

**Comment by Jayshua** [comment]
No problem, glad I could help!

**Pull request merged**